### ISSUE TITLE
Add definition to an existing label pigmentation phenotype

### ISSUE THREAD

**Issue by gaviganet** [obophenotype/mammalian-phenotype-ontology#2792]
>If this is a request for a new term, please provide us with the following information.  This request suggests a definition for an existing term. 

>1. Preferred term label (e.g., asplenia)

pigmentation phenotype

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

The observable characteristics of mammalian organisms that are manifested through coloring of the skin.
>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

mammalian phenotype (as is)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
KSU:jbengtson

**Comment by sbello** [comment]
Inclusion of 'skin' in the definition doesn't work as extracutaneous pigmentation is also included in this branch. I think if we change this to 'in any tissue or cell of the organism' this should work

**Comment by gaviganet** [comment]
Of course not just skin we will use this:
"The observable characteristics of mammalian organisms that are manifested
through coloring in any tissue or cell of the organism".
